model_utils/managers.py

```diff
@@ -260,11 +261,16 @@ def delete(self):
         self.update(is_removed=True)
 
 
-class SoftDeletableManager(models.Manager):
+class SoftDeletableQuerySet(SoftDeletableQuerySetMixin, QuerySet):
+    pass
+
+
+class SoftDeletableManagerMixin(object):
     """
     Manager that limits the queryset by default to show only not removed
     instances of model.
     """
+    use_for_related_fields = True
```

Comment:
This looks like an unrelated change that doesn't belong in this PR. It may be a warranted change, but it should have its own issue and PR and be discussed on its own merits. AFAICT it isn't mentioned in #249 or this PR description.